apps/researcher/src/messages/en/messages.json

```diff
@@ -120,7 +120,16 @@
     "transferredTo": "Transferred to",
     "location": "Location",
     "descriptionTitle": "Description",
-    "noData": "No provenance events found"
+    "noData": "No provenance events found",
+    "dataTableTitle": "Provenance data table",
+    "timelineTitle": "Provenance timeline",
+    "timelineDescription": "In the timeline visualization provenance items are plotted out by time. Click on a item focus that item.",
```

Comment:
1. Is the word `visualization` necessary? A timeline is a visualization?
1. Is the meaning of `provenance items` clear enough to users? An item, in my experience, is a collection item, e.g. an object in a museum. This data is about provenance events: what happened with an item/object on a certain date and location?
1. `Click on an item to focus on that item`?